modules/sdk-core/src/bitgo/tss/ecdsa/ecdsa.ts

```diff
@@ -369,8 +369,7 @@ export async function buildNShareFromAPIKeyShare(keyShare: ApiKeyShare): Promise
     publicShare: keyShare.publicShare,
     encryptedPrivateShare: keyShare.privateShare,
     n: keyShare.n ?? '', // this is not currently needed for key creation
-    // TODO (BG-65434) : make it as mandatory
-    privateShareProof: keyShare.privateShareProof ?? '',
+    privateShareProof: keyShare.privateShareProof,
```

Comment:
This change only makes it return undefined if it's not defined, doesn't make it mandatory as commented earlier. The type for `privateShareProof` is still optional. Is this the change you intended?

Reply:
yes that's good, so in wp we would be checking `if (privateShareProof)` .. it would throwing in errors anyways but in different errors.